src/controls/splash.js

```diff
@@ -17,34 +16,32 @@ const Splash = function Splash(options = {}) {
     url
   } = options;
 
+  const createModal = function createModal(modalContent) {
+    return Modal({
+      title,
+      content: modalContent,
+      cls,
+      target
+    });
+  };
+
   return Component({
     name: 'splash',
     onAdd(evt) {
       viewer = evt.target;
       target = viewer.getId();
       if (!title) title = defaultTitle;
```

Comment:
defaultTitle not set as defaultContent

Reply:
Not sure what you mean by this, care to clarify or give an example?

Reply:
Sorry, for some reason I missed the defaultTitle declaration.